app/src/pages/EthLockdropPage.tsx

```diff
@@ -0,0 +1,166 @@
+import {
+	IonLabel,
+	IonContent,
+	IonPage,
+	IonTitle,
+	IonToolbar,
+	IonHeader,
+	IonButton,
+	IonCol,
+	IonGrid,
+	IonRow,
+	IonItem,
+	IonInput,
+	IonCard,
+	IonCardContent
+} from '@ionic/react';
+import React, { useState } from 'react';
+
+import '../helpers/lockdrop/EthereumLockdrop';
+import { lockEthereum } from '../helpers/lockdrop/EthereumLockdrop';
+import { DropdownOption, OptionItem } from '../components/DropdownOption';
+
+const durations: OptionItem[] = [
+	{ label: '30 Days', value: 30 },
+	{ label: '100 Days', value: 100 },
+	{ label: '300 Days', value: 300 },
+	{ label: '1000 Days', value: 1000 }
+];
+
+const txTypes: OptionItem[] = [{ label: 'Metamask', value: 'metamask' }];
+
+const rates = [
```

Comment:
Currently, These rates are hidden from users.
We should explicitly show these rates to users, I think.

Reply:
resolved